src/components/ui/Animation.js

```diff
@@ -74,6 +74,7 @@ export default function animateChart(stx, animationParameters) {
     });
 
     stx.append('updateChartData', function (appendQuotes, chart, params) {
```

Comment:
Is object destruction a norm in the project, I mean, is it possible to use `{ noCreateDataSet }` instead of params in the function signature?

Reply:
@reza-fs Yes, we can use; there isn't not any rule for it.